opentracing/tracer.py

```diff
@@ -46,25 +79,42 @@ def start_span(self,
                    child_of=None,
                    references=None,
                    tags=None,
-                   start_time=None):
-        """Starts and returns a new Span representing a unit of work.
+                   start_time=None,
+                   ignore_active_span=False):
+        """Starts and returns a new Span representing a unit of work,
```

Comment:
Oh, yes, we can do that. Was wondering myself about that - so, in this case, I will add a new `start_manual_span` method AND mark the `start_span` as deprecated. Will put that update into the next iteration.